src/pages/index.tsx

```diff
@@ -175,50 +175,66 @@ const About: FC<{ data: AboutPageData }> = ({ data }) => {
             <Box className={classes.content}>{aboutData.description}</Box>
             <SectionTitle className={classes.title}>My skills</SectionTitle>
             <Box className={classes.content}>
-              <Typography variant="h6" className={classes.skillsHeader}>
-                Technologies
-              </Typography>
-              <Box className={classes.skills}>
-                {aboutData.socialMedia.technologies.map((item) => (
-                  <Skill key={`${item.technologyName}`} level={item.technologyValue}>
-                    {item.technologyName}
-                  </Skill>
-                ))}
-              </Box>
-              <Typography variant="h6" className={classes.skillsHeader}>
-                Tools
-              </Typography>
-              <Box className={classes.skills}>
-                {aboutData.socialMedia.tools.map((item) => (
-                  <Skill key={`${item.toolName}`} level={item.toolValue}>
-                    {item.toolName}
-                  </Skill>
-                ))}
-              </Box>
-              <Typography variant="h6" className={classes.skillsHeader}>
-                Other skills
-              </Typography>
-              <Box className={classes.skills}>
-                {aboutData.socialMedia.otherSkills.map((item) => (
-                  <Skill key={`${item.otherSkillName}`} level={item.otherSkillValue}>
-                    {item.otherSkillName}
-                  </Skill>
-                ))}
-              </Box>
-            </Box>
-            <SectionTitle className={classes.title}>Testimonials</SectionTitle>
-            <Box className={classes.testimonials}>
-              {aboutData.testimonials.map((item) => (
-                <Testimonial
-                  key={`${item.testimonialName}`}
-                  isMobile={isMobile}
-                  image={item.testimonialImage.publicURL}
-                  description={item.testimonialText}
-                  labelBold={item.testimonialName}
-                  label={item.testimonialJob}
-                />
-              ))}
+              {aboutData.socialMedia.technologies && aboutData.socialMedia.technologies.length > 0 && (
```

Comment:
Move it to a separate variable to describe its meaning (e.g. `hasTechnologies`)

Reply:
Alrighty 1/4 😄